llvm-external-projects/iree-dialects/lib/Transforms/ListenerGreedyPatternRewriteDriver.cpp

```diff
@@ -159,6 +163,16 @@ bool GreedyPatternRewriteDriver::simplify(MutableArrayRef<Region> regions) {
   };
 #endif
 
+  auto insertKnownConstant = [&](Operation *op) {
+    // Check for existing constants when populating the worklist. This avoids
```

Comment:
Why are we relying on a certain order in which constants are appearing in the IR? This seems brittle and could be broken by other transforms such as CSE.

Reply:
This updates the IREE version to the upstream version + least amount possible of Listener modifications.
Feel free to propose changes upstream, but here is not the place.